src/graphics/SVG/Radar/ObstacleRenderer.php

```diff
@@ -83,4 +84,26 @@ protected function objectToSvgNode(string $cls): SVGNode
 
         return $svg;
     }
+
+    protected function meshToSVGNode(string $class): SVGNode
+    {
+        $svg = new $class();
+        $svgPolygon = SVGPolygon::class;
+        foreach ($this->obstacle->getFaces() as $meshFace)
+        {
+            $mesh = new $svgPolygon(
+                $meshFace->getVertices(),
+            );
+            $svg->addChild($mesh);
+        }
```

Comment:
```suggestion
        foreach ($this->obstacle->getFaces() as $meshFace)
        {
            $mesh = new SVGPolygon($meshFace->getVertices());
            $svg->addChild($mesh);
        }
```

You don't need to store the `SVGPolygon` class name into a variable. That's done for `$class` because it's figured out dynamically at run time. But we know that each time this function is called, it'll always be an `SVGPolygon`